topology_test.go

```diff
@@ -22,6 +22,9 @@ func TestController_AddInstance(t *testing.T) {
 	t.Run("no such replicaset", func(t *testing.T) {
 		router := Router{
 			idToReplicaset: map[uuid.UUID]*Replicaset{},
+			cfg: Config{
+				Loggerf: &emptyLogger{},
```

Comment:
We can use here 'emptyLogfProvider' instead of '&emptyLogger{}'
```golang
Loggerf: emptyLogfProvider,
```

Reply:
it is not implemented yet, I think it’s not critical